src/game/GameProcMain.cpp

```diff
@@ -4524,6 +4531,36 @@ bool CGameProcMain::CommandToggleUISkillTree() {
     return bNeedOpen;
 }
 
+bool CGameProcMain::CommandToggleUINotice() {
+    bool bNeedOpen = !m_pUINotice->IsVisible();
+
+    if (m_pSubProcPerTrade->m_ePerTradeState != PER_TRADE_STATE_NONE) {
+        return bNeedOpen;
+    }
```

Comment:
Hi @xGuTeK, I noticed you added this, but it has no references. Do you remember or know what key binding opening and closing the notice UI was set officially?

Reply:
What you mean it has no references?

I don't remember, I have even tried to find key binding on Janek Server (1298) but without success but he has customizabled client so he may deleted this maybe or key binding for this not exists

Reply:
> What you mean it has no references?
> 
> I don't remember, I have even tried to find key binding on Janek Server (1298) but without success but he has customizabled client so he may deleted this maybe or key binding for this not exists

btw I have a 1298 server running if you want to use it for testing